src/lib/template_style/_form.scss

```diff
@@ -103,8 +103,8 @@
   }
   input[type='checkbox']:focus + label::before,
   input[type='checkbox']:focus + span::before,
-  input[type='radio']:focus:checked + label::before,
-  input[type='radio']:focus:checked + span::before,
+  input[type='radio']:focus + label::before,
+  input[type='radio']:focus + span::before,
```

Comment:
Small style issue, radio should always have this drop shadow, not just when checked.